circle.yml

```diff
@@ -10,7 +10,6 @@ dependencies:
     - echo "memory_limit=520M" > ~/.phpenv/versions/$(phpenv global)/etc/conf.d/memlimit.ini
     - if [ "$CIRCLE_NODE_INDEX" = "1" ]; then ./tests/bin/setup.mysql.sh; fi
     - if [ "$CIRCLE_NODE_INDEX" = "2" ]; then ./tests/bin/setup.pgsql.sh; fi
-    - if [ "$CIRCLE_NODE_INDEX" = "3" ]; then ./tests/bin/setup.sqlite.sh; fi
```

Comment:
https://circleci.com/gh/propelorm/Propel2/275
```
if [ "$CIRCLE_NODE_INDEX" = "3" ]; then ./tests/bin/setup.sqlite.sh; fi
./tests/bin/setup.sqlite.sh: 9: ./tests/bin/setup.sqlite.sh: [[: not found 
```
This script does not work for a long time, and the tests are working is good.
I figured if it didn't work, and the tests work, then I can delete it)